libs/core/device/actions/unlock-device.action.ts

```diff
@@ -27,6 +28,8 @@ export const unlockDevice = createAsyncThunk<
     )
   }
 
+  await delay(500)
```

Comment:
Is this really a solution or rather workaround?

Reply:
this is the issue from the API, API can't change before state in DB between requests